### PR TITLE
Add theta and theta off calculations

### PR DESCRIPTION
**Pull request by maxnoe**
So we are independent of the imprecise coordinate trafos in FACT-Tools

### CODE REVIEW

File: fact/analysis/source.py [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+    source_x: float or astropy.units.Quantity
+        x position of the wanted source position
+    source_y: float or astropy.units.Quantity
+        y position of the wanted source position
+    n_off: int
+        number of off positions to calculate
+
+    Returns
+    -------
+    x_off: float or or astropy.units.Quantity
```
Comment (jebuss) [inline fact/analysis/source.py]: why two `or`

Comment (maxnoe) [reply fact/analysis/source.py]: because of security reasons

File: fact/analysis/source.py [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+def calc_theta_camera(
+        source_x_prediction,
+        source_y_prediction,
+        source_zd,
+        source_az,
+        zd_pointing,
+        az_pointing):
+    '''
+    Calculate the angular distance between reconstructed source
+    position and assumed source position, where the
```
Comment (jebuss) [inline fact/analysis/source.py]: `assumed`to `predicted`

**Review by jebuss — APPROVED** [review]
noice

**Pull request merged**